### PR TITLE
Development has

### PR DESCRIPTION
**Pull request by hasanhameedtp**
App activate/deactivate, update, delete

### CODE REVIEW

File: crust/controllers/application.js [hunk 1]
```diff
@@ -4,27 +4,46 @@ const applicationService = require('../../lib/facade/application.js');
...
 };
 
 /** Controller to resolve app listing request*/
 function getAppListing(req, res, next) {
   applicationService.getAppListing(req.body)
     .then(function (response) {
-      return res.status(200).json(response);
+      return res.status(200).json({
+        data: response,
+        meta: [{length: response.length}]
```
Comment (faizulhaque-tp) [inline crust/controllers/application.js:18]: any reason to make meta as array of object rather then object?

File: crust/controllers/application.js [hunk 2]
```diff
@@ -4,27 +4,46 @@ const applicationService = require('../../lib/facade/application.js');
...
+      });
     }).catch(next);
 }
 
 function createApp(req, res, next) {
   applicationService.createApp(req.body)
     .then((response) => {
-      res.status(201).json(response);
+      res.status(201).json({message: "App successfully created"});
```
Comment (faizulhaque-tp) [inline crust/controllers/application.js]: its better to return newly created object.

File: lib/facade/application.js [hunk 3]
```diff
@@ -1,24 +1,21 @@
 'use strict';
-const appRepo = require('../repositories/app');
 const validator = require('../validation');
 const helper = require('../helpers/commonHelper');
 const rand = require("random-key");
+const App = require('../models/App');
```
Comment (faizulhaque-tp) [inline lib/facade/application.js:6]: in-consistent naming convention.

App should be app.

File: crust/controllers/application.js (hunk 1 above)
Comment (hasanhameedtp) [inline crust/controllers/application.js:18]: @raja-sami-10p

Comment (raja-sami-10p) [inline crust/controllers/application.js:18]: @hasanhameedtp change it to object, I looked up json response standards.

**Pull request merged**